api/admin/controller.py

```diff
@@ -540,8 +544,8 @@ def refresh_metadata(self, identifier_type, identifier, provider=None):
 
         if record.exception:
             # There was a coverage failure.
-            if (isinstance(record.exception, int)
-                and record.exception in [201, 202]):
+            if (str(record.exception).startswith("201") or
+                str(record.exception).startswith("202")):
```

Comment:
I'm not sure about this change. I was getting "202: I'm working to locate a source for this identifier." instead of an int when I tested it.

Reply:
I think the new behavior is right. `record.exception` is created from `OPDSMessage.status_code` and `OPDSMessage.message`, in `OPDSImporter.coveragefailure_from_message`